addons/website_sale_slides/views/website_slides_templates.xml

```diff
@@ -32,6 +32,20 @@
     </xpath>
 </template>
 
+<template name="Slide template for a training channel" id="course_slides_list_slide_buy_course" inherit_id="website_slides.course_slides_list_slide">
+    <xpath expr="//span[@name='preview_badge_admin']" position="before">
+        <span t-if="channel.enroll == 'payment'" t-att-data-slide-id="slide.id" t-attf-class="o_wslides_js_slide_toggle_is_preview badge #{'badge-success' if slide.is_preview else 'badge-light badge-hide border'} font-weight-normal px-2 py-1 m-1">Free Preview</span>
+    </xpath>
+    <xpath expr="//span[@name='preview_badge_admin']" position="attributes">
+        <attribute name="t-else">1</attribute>
+    </xpath>
+    <xpath expr="//span[@name='preview_badge']" position="before">
+        <span t-if="channel.enroll == 'payment'" class="badge badge-success font-weight-normal px-2 py-1 m-1">Free Preview</span>
+    </xpath>
+    <xpath expr="//span[@name='preview_badge']" position="attributes">
+        <attribute name="t-else">1</attribute>
+    </xpath>
```

Comment:
why not a replace, if it's just to change 2 words in the button ?
```
    <xpath expr="//span[@name='preview_badge']" position="replace">
        <span class="badge badge-success font-weight-normal px-2 py-1 m-1"><t esc="'The Old Brol' if channel.enroll == 'payment' else 'Free Preview'"/></span>
    </xpath>
```
or something like that

Reply:
quid about translation?